sjb/hack/determine_install_upgrade_version.py

```diff
@@ -114,7 +114,10 @@ def sort_pkgs(available_pkgs):
 	# listing of origin or docker pkgs will be disabled, based on the enabled excluder.
 	yb.conf.disable_excludes = ["all"]
 	sys.stdout = old_stdout
-	available_pkgs = rpmutils.unique(yb.doPackageLists('available', patterns=[pkg_name], showdups=True).available)
+
+	generic_holder = yb.doPackageLists('all', patterns=[pkg_name], showdups=True)
+	available_pkgs = rpmutils.unique(generic_holder.available + generic_holder.installed)
```

Comment:
I just meant it might be easier to look at the output from debugging info if we took a debugging log file and put it all in there -- rather than looking through the job log. If this is outputting a lot of stuff or if you expect to add more debug info maybe that would be appropriate? Your call.